hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -835,13 +830,11 @@ public HoodieMetadataValidationContext(
                     metaClient.getHadoopConf(),
                     new Path(new Path(metaClient.getBasePath(), partitionPath), filename),
                     allColumnNameList).stream())
-            .map(rangeMetadata -> HoodieColumnRangeMetadata.create(
+            .map(rangeMetadata -> HoodieColumnRangeMetadata.<Comparable>create(
```

Comment:
`ParquetUtils().readRangeFromParquetMetadata` already returns `List<HoodieColumnRangeMetadata<Comparable>>` so why recreate? Can't we directly sort the stream on #L832 ?

Reply:
Agreed, i overlooked that we can skip that altogether